index.html

```diff
@@ -53,5 +53,151 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="card-languages-list">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="btn-block btn-primary">
+            See Project
+          </button>
+        </div>
+        <div class="card">
+          <h1 class="card-heading">Professional Art Printing Data</h1>
+          <p class="card-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="card-languages-list">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="btn-block btn-primary">
+            See Project
+          </button>
+        </div>
+        <div class="card">
+          <h1 class="card-heading">Professional Art Printing Data</h1>
+          <p class="card-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="card-languages-list">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="btn-block btn-primary">
+            See Project
+          </button>
+        </div>
+      </div>
+    </section>
+    <section class="about-section">
+      <h1 class="about-heading">About me</h1>
+      <p class="about-description">
+        Hello I'm a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hesitate to contact me.
+      </p>
+      <div class="btn-left">
+        <button type="button" class="btn-primary">Get My Resume</button>
+      </div>
+      <div class="about-image"></div>
+      <hr />
+      <ul class="about-list">
+        <li class="language-list">Language</li>
+        <li class="list-style-none">Javascript</li>
+        <li class="list-style-none">Ruby</li>
+        <li class="list-style-none">Html</li>
+        <li class="list-style-none">Css</li>
+        <li class="frameworks-list">Frameworks</li>
+        <li class="list-style-none">Bootstrap</li>
+        <li class="list-style-none">Ruby on Rails</li>
+        <li class="list-style-none">RSPec</li>
+        <li class="list-style-none">CapyBara</li>
+        <li class="list-style-none">Selenium</li>
+        <li class="skills-list">Skills</li>
+        <li class="list-style-none">Codekit</li>
+        <li class="list-style-none">Github</li>
+        <li class="list-style-none">Codepen</li>
+        <li class="list-style-none">Gitlab</li>
+        <li class="list-style-none">Terminal</li>
+      </ul>
```

Comment:
- [ ] I really appreciate your effort in trying to match your design with the design given 👏. In addition, great job for you have used list tags here 👏. What I would like you to improve on is that, as you see the technology categories("Languages", "Frameworks", "Skills") are list items and it is right but their belongings(for example; Languages list contains "Ruby", "Javascript", etc ) should be nested inside them. Let me show you what I would like you to do by example;
```bash
<ul>
     <li>Languages
            <ul>
                 <li>Javascript</li>
                <li>Ruby</li>
                <li>Html</li>
               <li>CSS</li>
            </ul>
     </li>
</ul>
```
This is what I would like you to follow because it is **more logical** and it is among the requirements. It is referred to as **Nesting list into another list**. For more information, please read this [documentation](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/) 💡 😃